.github/actions/minus-one/src/index.mjs

```diff
@@ -0,0 +1,44 @@
+// @ts-check
+import { context } from '@actions/github'
+import { info, setFailed } from '@actions/core'
+import { graphql } from '@octokit/graphql'
+
+const offTopicComments = ['+1', '👍', 'same issue', 'any updates']
```

Comment:
refined github https://github.com/refined-github/refined-github/blob/c864a20b57bb433aaf3952f88d83c9fc481ae6ff/source/helpers/is-low-quality-comment.ts#L3C44-L3C44 has a regex to find some more commonly used offTopic comments